File: code-coaches/easy/neverland/Neverland.cs
using System;

public class Neverland
{
    static void Main(string[] args)
    {
        int initialAge = int.Parse(Console.ReadLine());
        int yearsPassed = int.Parse(Console.ReadLine());
        int finalAge = initialAge + yearsPassed;

        Console.WriteLine($"My twin is {finalAge} years old and they are {yearsPassed} years older than me");
    }
}

File: code-coaches/easy/neverland/neverland.rb
def main
  initial_age = gets.chomp.to_i
  years_passed = gets.chomp.to_i
  final_age = initial_age + years_passed
  puts "My twin is #{final_age} years old and they are #{years_passed} years older than me"
end

if __FILE__ == $0
  main
end
